TeamCode/src/main/java/suitbots/Robot.java

```diff
@@ -17,12 +17,13 @@
 
 public class Robot {
     //Yo necesito dormir.
+    private int gyrochecker = 0;
     private double lastG;
-    private BNO055IMU gyro;
+    private BNO055IMU imu;
     private DcMotor lf, lr, rf, rr;
     private ColorSensor lineDetector;
     public Robot(HardwareMap h) {
-        gyro = h.get(BNO055IMU.class, "gyro");
+        imu = h.get(BNO055IMU.class, "gyro");
```

Comment:
The difference in name is in the configuration. So the variable name is fine, but the configuration name between this and the calibrator should match. (This is currently `"gyro"` and the calibration example `"imu"`.)

Reply:
Should we keep the configuration name gyro or imu?

Reply:
Either one is fine just so long as they're the same here and in the calibration example. Probably easiest to go with `"imu"`.